From 8dee7fb2a435b2b6f5d452bd7a7bb30f3f3ccef0 Mon Sep 17 00:00:00 2001
From: Jouni Malinen <j@w1.fi>
Date: Sun, 29 Jan 2012 14:20:41 +0200
Subject: [PATCH 09/73] Install only the binaries into BINDIR

There is no point in installing *.service files into BINDIR.

Signed-hostap: Jouni Malinen <j@w1.fi>
(cherry picked from commit ea5cb06ef907d4453c75d700bbf0322192990cb8)
---
 wpa_supplicant/Makefile |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

--- a/wpa_supplicant/Makefile
+++ b/wpa_supplicant/Makefile
@@ -15,12 +15,13 @@ CFLAGS += -I../src/utils
 
 -include .config
 
-ALL=wpa_supplicant wpa_cli
+BINALL=wpa_supplicant wpa_cli
 
 ifndef CONFIG_NO_WPA_PASSPHRASE
-ALL += wpa_passphrase
+BINALL += wpa_passphrase
 endif
 
+ALL = $(BINALL)
 ALL += systemd/wpa_supplicant.service
 ALL += systemd/wpa_supplicant@.service
 ALL += systemd/wpa_supplicant-nl80211@.service
@@ -51,7 +52,7 @@ mkconfig:
 $(DESTDIR)$(BINDIR)/%: %
 	install -D $(<) $(@)
 
-install: $(addprefix $(DESTDIR)$(BINDIR)/,$(ALL))
+install: $(addprefix $(DESTDIR)$(BINDIR)/,$(BINALL))
 	$(MAKE) -C ../src install
 
 OBJS = config.o
